topic_folders/policies/immediate-response-checklist.md

```diff
@@ -0,0 +1,74 @@
+
+### Teaching Workshops
+The suspended member will, for the duration of their suspension:
+- [ ] Be asked not to sign up for workshops on the Instructor Spreadsheet.
+- [ ] Not be able to participate as a helper at any workshops.
+- [ ] Be removed from Eventbrite login access.
+- [ ] Not be allowed to plan self-organized workshops.
+- [ ] Not be allowed to request or host centrally coordinated workshops.
+- [ ] Not be listed as an instructor on the “Instructors” webpage.
+- [ ] Be removed from the instructors email list.
+
+### Organizing Workshops 
+If the suspended member is a Workshop Administrator or serves in another role whereby they organize workshops for The Carpentries, for the duration of their suspension they will not have access to:
+- [ ] LastPass
+- [ ] AMY
+- [ ] Regional Email
+- [ ] Asana
+- [ ] HelpScout
+- [ ] Zoom
+- [ ] Workshop Administrator’s Slack Channel
+- [ ] Workshop Administrator’s Email List
+
+### Instructor Training
+The suspended member will, for the duration of their suspension, be removed from the “Our Trainers” web page. Additionally, the suspended member will not be allowed to: 
+- [ ] Teach Instructor Training or lead teaching demonstration sessions. The Deputy Director of Instructor Training (DDIT) will check the teaching demo Etherpad to ensure that the suspended member is not signed up to lead teaching demonstrations and will find a replacement Trainer as needed. 
+- [ ] Join Trainer meetings. The DDIT will check the meeting Etherpad, and if the suspended member is signed up to attend, will email them if needed to confirm they will not be in attendance. 
+- [ ] Post pull requests (PRs), issues, or comments to the Instructor Training GitHub repository. Write or administrative access to repositories will be removed for the duration of the suspension. 
+- [ ] Post to the Trainers Slack channel or mailing lists. The suspended member will be removed from these lists for the duration of their suspension. 
+- [ ] Participate in review of applications for open instructor training or Trainer training. 
+Participate as a learner in instructor training.
+
+### Trainer Training
+The suspended member will not be allowed to participate in Trainer Training (if they are currently enrolled) and will not be accepted for Trainer Training (if they apply during the suspension period). Additionally, they will not be allowed to shadow a current Carpentries Trainer. They may apply or reapply for future rounds of Trainer Training. 
+
+### Member Organisation and Local Activities
+The suspended member shall not be involved in officially branded Carpentries workshops or activities at the member or local site for the duration of the suspension. We expect the suspended member to assert that they will be able to carry out this suspension accordingly and to notify the CoC committee if this suspension will interfere with their expected employment duties. 
```

Comment:
Again something I think should probably be adjusted to level of seriousness for issues under review.

Reply:
This is also getting into some strange territory where it reads like we are expecting them to excuse themselves.  Perhaps, like some of my other comments, it could be stated that the committee will make a decision depending on severity and risk that they will do one of:  contact local organizers to make them away of the person's suspension (most severe, and likely quite rare as this would publicize their CoC violation status), direct the suspended person to decline or cancel any workshop participation during their suspension period, or not make this request of them.

Reply:
Agreed. Case-by-case basis. The intro language includes a clause about the severity of the situation, so we're all set.